crates/pallet-domains/src/domain_registry.rs

```diff
@@ -147,6 +142,7 @@ pub(crate) fn do_instantiate_domain<T: Config>(
         created_at,
         genesis_receipt_hash,
         domain_config,
+        raw_genesis_config,
```

Comment:
Why do we need to pass raw genesis to the DomainObject here?

Reply:
Because all the domain's genesis config (including the genesis domain) is maintained in the consensus chain, when a domain node is started, the bootstrapper will first fetch the related data from the consensus chain and these data to construct the domain's genesis config.

In the main branch, these data are the `domain_id` and the runtime code, in this PR there is an additional `raw_genesis_config`.